tools/lsp/main.rs

```diff
@@ -370,56 +387,75 @@ async fn handle_notification(req: lsp_server::Notification, ctx: &Rc<Context>) -
                 Some(params.text_document.version),
                 &mut ctx.document_cache.borrow_mut(),
             )
-            .await?;
-        }
-        DidChangeConfiguration::METHOD => {
-            load_configuration(ctx).await?;
+            .await
         }
+        DidChangeConfiguration::METHOD => load_configuration(ctx).await,
 
         #[cfg(any(feature = "preview-builtin", feature = "preview-external"))]
         "slint/showPreview" => {
-            language::show_preview_command(
-                req.params.as_array().map_or(&[], |x| x.as_slice()),
-                ctx,
-            )?;
+            language::show_preview_command(req.params.as_array().map_or(&[], |x| x.as_slice()), ctx)
         }
 
+        // Messages from the WASM preview come in as notifications sent by the "editor":
         #[cfg(all(feature = "preview-external", feature = "preview-engine"))]
         "slint/preview_to_lsp" => {
             use common::PreviewToLspMessage as M;
             let params: M = serde_json::from_value(req.params)?;
```

Comment:
That use statement is not very useful for just one use.

This is probably not needed at all because of inference
```suggestion

            let params = serde_json::from_value(req.params)?;
```